### PR TITLE
added topics in readme for git theory

### PR DESCRIPTION
**Pull request by Saddamkhanlavani**
Added day1/2/3/4 topics for git-theory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+    Class - 3: Git Advanced practical
+
+    Git Commands:
+        git fetch, git stash, git pop, git pull, git merge, git rebase, git log.
+
+    Topics: 
+        Rebase vs Merge, Pull vs fetch.
+
+        Class - 4:
```
Comment (IshaaqhShaik) [inline README.md:47]: Please correct the indentation here.

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+        Importance of using branching
+
+        add . vs add filename, checkout -b vs checkout branch vs checkout ., 
+
+Task: Getting handson with the commands discussed.
+
+    Create a branch on the repo provided add the changes.
+
+    Class - 3: Git Advanced practical
```
Comment (IshaaqhShaik) [inline README.md:39]: please correct the indentation here.

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Saddamkhanlavani) [reply README.md:47]: corrected the indentation.

File: README.md (hunk 2 above)
Comment (Saddamkhanlavani) [reply README.md:39]: Completed

File: README.md (hunk 1 above)
Comment (Saddamkhanlavani) [reply README.md]: Now corrected the indentation.

**Review by IshaaqhShaik — APPROVED** [review]

**Pull request merged**